forge-gui/res/cardsfolder/upcoming/the_war_games.txt

```diff
@@ -0,0 +1,12 @@
+Name:The War Games
+ManaCost:2 W W
+Types:Enchantment Saga
+K:Saga:4:DBTokens,DBPutCounterAll,DBPutCounterAll,DBMayExile
+SVar:DBTokens:DB$ Token | TokenAmount$ 3 | TokenTapped$ True | TokenScript$ w_1_1_warrior | TokenOwner$ Player | RememberTokens$ True | SubAbility$ DBGoad | SpellDescription$ Each player creates three tapped 1/1 white Warrior creature tokens. The tokens are goaded for as long as CARDNAME remains on the battlefield.
+SVar:DBGoad:DB$ Goad | Defined$ Remembered | Duration$ UntilHostLeavesPlay | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:DBPutCounterAll:DB$ PutCounterAll | ValidCards$ Creature.Warrior | CounterType$ P1P1 | CounterNum$ 1 | SpellDescription$ Put a +1/+1 counter on each Warrior creature.
+SVar:DBMayExile:AB$ ImmediateTrigger | Cost$ Exile<1/Creature.nonToken> | Execute$ TrigExileAll | SpellDescription$ You may exile a nontoken creature you control. When you do, exile all Warriors.

```

Comment:
Moved the description onto the following line and hooked the ImmediateTrigger up directly. Is there anything more I should do that you didn't mention previously?

Reply:
no, that looks fine
I'm not sure if you were already a contributor a few years ago but the average card can be quite complex these days
Reflexive triggers are at least easy to recognize - "when you do" is always one. that does not cover 100% of the cases 

But always happy to clarify

Reply:
**D**raw**b**ack is the default, but by convention does not support having a Cost$
Depending on the scenario it can make a difference if you write it with one (instead of effect) as long as the engine supports it (and usually helps with readability too)